### PR TITLE
Prepare version 0.9.2

### PR DESCRIPTION
**Pull request by tunetheweb**
Prepares for the v0.9.2 release

Please comment below if you have an imminent Pull Request incoming that you would like to see included in this release.

I'll probably merge this Sunday night or Monday morning to give some time to anyone working on changes.

### CODE REVIEW

**Comment by sti0** [comment]
Would be great if we could add https://github.com/sqlfluff/sqlfluff/pull/2407 to this release.

**Comment by jpy-git** [comment]
Would be good to get #2404 in 😄

**Comment by rpr-ableton** [comment]
If #2417 could make it in the release, it would be great :raised_hands:

**Comment by chwiese** [comment]
@tunetheweb  #2421 is a small PR that could be included, if anybody has time to review it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,121 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
-## [0.9.1] - 2022-01-08]
+
+## [0.9.2] - 2022-01-24
+
+## Highlights
+
+We are please to include over 80 fixes in this release, and welcome seven new contributors.
```
Comment (kdw2126) [inline CHANGELOG.md:17]: Shouldn't this be "We are pleased to include over 80 fixes in this release and welcome seven new contributors"?  Sorry to nitpick -- love everything you do for the library @tunetheweb!

Comment (tunetheweb) [reply CHANGELOG.md]: Thanks! - fixed.

**Comment by tunetheweb** [comment]
Synced up the latest PRs. We're at 95 PRs in this release which is amazing in two week! We've a few more in the pipeline and am sure we can make 100.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,7 +9,139 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+## [0.9.2] - 2022-01-24
+
+## Highlights
+
+We are pleased to include over 95 fixes in this release, and welcome seven new contributors.
+
+Major changes include:
+
+* Initial Oracle support (note: SQL but PL/SQL)
```
Comment (chwiese) [inline CHANGELOG.md:21]:
```suggestion
* Initial Oracle support (note: SQL, but not PL/SQL)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,7 +9,139 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+* Fix more dbt 1.0.0 connection issue
```
Comment (chwiese) [inline CHANGELOG.md:22]:
```suggestion
* Fix more dbt 1.0.0 connection issues
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,7 +9,139 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+* Improved configureation documentation
```
Comment (chwiese) [inline CHANGELOG.md:23]:
```suggestion
* Improved configuration documentation
```

**Review by chwiese — COMMENTED** [review]
I spotted a few minor typos

**Comment by tunetheweb** [comment]
Thanks @chwiese ! All accepted.

**Review by barrywhart — APPROVED** [review]

**Comment by tunetheweb** [comment]
OK I'm calling it now. Will merge once tests complete, and then release.

110 changes in 15 days - some effort everyone!!!

**Pull request merged**